p4-16/spec/grammar.mdk

```diff
@@ -178,14 +178,22 @@ keysetExpression
     ;
 
 tupleKeysetExpression
-    : '(' simpleKeysetExpression ',' simpleExpressionList ')'
+    : "(" simpleKeysetExpression "," simpleExpressionList ")"
+    | "(" reducedSimpleKeysetExpression ")"
```

Comment:
Why not use simpleKeysetExpression here?  What is the need for introducing reducedSimpleKeysetExpression?

Reply:
reducedKeysetExpression is missing a case, `expression`